ui/src/test/java/com/hyperwallet/android/ui/repository/UserRepositoryImplTest.java

```diff
@@ -0,0 +1,204 @@
+                .profileType(INDIVIDUAL)
+                .firstName("Some")
+                .lastName("Guy")
+                .dateOfBirth("1991-01-01")
+                .email("testUser@hyperwallet.com")
+                .addressLine1("575 Market Street")
+                .city("San Francisco")
+                .stateProvince("CA")
+                .country("US")
+                .postalCode("94105")
+                .language("en")
+                .programToken("prg-83836cdf-2ce2-4696-8bc5-f1b86077238c")
+                .build();
+
+        doAnswer(new Answer() {
+            @Override
+            public Object answer(InvocationOnMock invocation) {
+                HyperwalletListener listener = (HyperwalletListener) invocation.getArguments()[0];
+                listener.onSuccess(user);
+                return listener;
+            }
+        }).when(mHyperwallet).getUser(ArgumentMatchers.<HyperwalletListener<HyperwalletUser>>any());
+        UserRepository.LoadUserCallback mockCallback = mock(UserRepository.LoadUserCallback.class);
+
+        mUserRepository.loadUser(mockCallback);
+
+        verify(mockCallback).onUserLoaded(mUserCaptor.capture());
+        verify(mockCallback, never()).onError(any(HyperwalletErrors.class));
+
+        HyperwalletUser resultUser = mUserCaptor.getValue();
+        assertThat(resultUser.getField(TOKEN), is("usr-f9154016-94e8-4686-a840-075688ac07b5"));
+        assertThat(resultUser.getField(STATUS), is(PRE_ACTIVATED));
+        assertThat(resultUser.getField(VERIFICATION_STATUS), is(NOT_REQUIRED));
+        assertThat(resultUser.getField(CREATED_ON), is("2017-10-30T22:15:45"));
+        assertThat(resultUser.getField(CLIENT_USER_ID), is("123456"));
+        assertThat(resultUser.getField(PROFILE_TYPE), is(INDIVIDUAL));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.FIRST_NAME), is("Some"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.LAST_NAME), is("Guy"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.DATE_OF_BIRTH), is("1991-01-01"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.EMAIL), is("testUser@hyperwallet.com"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.ADDRESS_LINE_1), is("575 Market Street"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.CITY), is("San Francisco"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.STATE_PROVINCE), is("CA"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.COUNTRY), is("US"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.POSTAL_CODE), is("94105"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.LANGUAGE), is("en"));
+        assertThat(resultUser.getField(HyperwalletUser.UserFields.PROGRAM_TOKEN),
+                is("prg-83836cdf-2ce2-4696-8bc5-f1b86077238c"));
+    }
+
+    @Test
+    public void testLoadUser_returnEmptyUserData() {
```

Comment:
please rename it to testLoadUser_returnsNoUser

Reply:
@fmattos-hw Thank you, missed, done.